tests/model/test_consolidated_issue.py

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright 2024 ABSA Group Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from github.Issue import Issue
+
+from living_documentation_generator.model.consolidated_issue import ConsolidatedIssue
+from living_documentation_generator.model.project_status import ProjectStatus
+
+
+# update_with_project_data
+
+
+# THIS MIGHT NOT BE NEEDED, SINCE WE TEST ASSIGN AND APPEND
```

Comment:
You are right. An extra test is not needed here. The coverage will be reached from the test on a higher level.

Reply:
Commit: [e760a98](https://github.com/AbsaOSS/living-doc-generator/pull/42/commits/e760a988275b27be90f0b64acddf8118271c7bfa).